src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/ioc/FeesAndPaymentsAreUp.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.ioc;
+
+import org.springframework.boot.autoconfigure.condition.AllNestedConditions;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+
+public class FeesAndPaymentsAreUp extends AllNestedConditions {
```

Comment:
`FeesAndPaymentsAreUp` name is little misleading. It suggests that it is the health check which is not correct.

Reply:
👍